modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsRelationshipsDisplayContext.java

```diff
@@ -210,6 +212,19 @@ public String getRESTContextPath(ObjectDefinition objectDefinition) {
 		return jaxRsApplicationDescriptor.getRESTContextPath();
 	}
 
+	public boolean isAddress() {
+		ObjectDefinition objectDefinition = getObjectDefinition();
+
+		if (objectDefinition.isUnmodifiableSystemObject() &&
+			StringUtil.equals(
+				objectDefinition.getName(), Address.class.getSimpleName())) {
```

Comment:
@murilostodolni can we use the `objectDefinition.getClassName()` instead?

Reply:
Yes, we can. I'm changing this.